packages/cli/src/commands/generate/service/templates/service.ts.template

```diff
@@ -0,0 +1,48 @@
+import { db } from 'src/lib/db'
+<%if (crud) { %>import {
+  ${singularPascalName}WhereUniqueInput,
+  ${singularPascalName}CreateInput,
+  ${singularPascalName}UpdateInput,
+} from '@prisma/client'
+<% } %>
+export const ${pluralCamelName} = () => {
+  return db.${singularCamelName}.findMany()
+}<% if (crud) { %>
+
+export const ${singularCamelName} = ({ id }: ${singularPascalName}WhereUniqueInput) => {
+  return db.${singularCamelName}.findOne({
+    where: { id },
+  })
+}
+
+interface Create${singularPascalName}Args {
+  input: ${singularPascalName}CreateInput
+}
+
+export const create${singularPascalName} = ({ input }: Create${singularPascalName}Args) => {
+  return db.${singularCamelName}.create({
+    data: input,
+  })
+}
+
+interface Update${singularPascalName}Args {
+  where: ${singularPascalName}WhereUniqueInput
+  input: ${singularPascalName}UpdateInput
+}
+
+export const update${singularPascalName} = ({ id, input }: Update${singularPascalName}Args) => {
+  return db.${singularCamelName}.update({
+    data: input,
+    where: { id },
+  })
+}
+
+export const delete${singularPascalName} = ({ id }: ${singularPascalName}WhereUniqueInput) => {
+  return db.${singularCamelName}.delete({
+    where: { id },
+  })
+}<% } %><% if (relations.length) { %>
+
+export const ${singularPascalName} = {<% relations.forEach(relation => { %>
+  ${relation}: (_obj, { root }) => db.${singularCamelName}.findOne({ where: { id: root.id } }).${relation}(),<% }) %>
```

Comment:
`root` argument can we do something about it?

Reply:
![image](https://user-images.githubusercontent.com/35949/82123736-e3a73300-9768-11ea-90c6-efaf53fe67dc.png)

Does this look about right?